src/algorithms/reco/JetReconstruction.cc

```diff
@@ -55,6 +56,19 @@ namespace eicrecon {
     // Choose jet definition based on no. of parameters
     switch (m_mapJetAlgo[m_cfg.jetAlgo]) {
 
+      // contributed algorithms
+      case JetAlgorithm::plugin_algorithm:
+
+        // expand to other algorithms as required
+        if(m_cfg.jetContribAlgo != "Centauro"){
+          m_log->error(" Unknown contributed FastJet algorithm \"{}\" specified!", m_cfg.jetContribAlgo);
+          throw JException("Invalid contributed FastJet algorithm");
+        }
+        else{
+          m_jet_def = std::make_unique<JetDefinition>(new contrib::CentauroPlugin(m_cfg.rJet));
+        }
```

Comment:
```suggestion
        // expand to other algorithms as required
        if(m_cfg.jetContribAlgo == "Centauro"){
          m_jet_def = std::make_unique<JetDefinition>(new contrib::CentauroPlugin(m_cfg.rJet));
        }
        else {
          m_log->error(" Unknown contributed FastJet algorithm \"{}\" specified!", m_cfg.jetContribAlgo);
          throw JException("Invalid contributed FastJet algorithm");
        }
```

This makes it a little easier to extend if we want to add another contributed algorithm!